pages/index.js

```diff
@@ -183,6 +157,7 @@ const STYLES_H1 = css`
 
 const STYLES_H2 = css`
   font-size: ${Constants.typescale.lvl3};
+  font-weight: 500;
```

Comment:
Don't use font-weight unless you're using a system font that has the weightings, we use inter and use a specific woff per font-weight.